app/src/ui/clone-repository/cloneable-repository-filter-list.tsx

```diff
@@ -157,26 +159,34 @@ export class CloneableRepositoryFilterList extends React.PureComponent<ICloneabl
     const { repositories, account, selectedItem } = this.props
 
     const groups = this.getRepositoryGroups(repositories, account.login)
+    const groupAriaLabel = (group: number) => {
+      const groupIdentifier = groups[group].identifier
+      return groupIdentifier === YourRepositoriesIdentifier
+        ? this.getYourRepositoriesLabel()
+        : groupIdentifier
+    }
+
     const selectedListItem = this.getSelectedListItem(groups, selectedItem)
+    const ListComponent = enableSectionList() ? SectionFilterList : FilterList
```

Comment:
This is slick. Would not have thought that was thing tho it makes perfect sense.

Reply:
Yes! Although I wasn't able to type the props properly 😞